ckanext/who_afro/templates/user/read_base.html

```diff
@@ -0,0 +1,26 @@
+{% ckan_extends %}
+
+
+{% block breadcrumb %}
+    {% if self.breadcrumb_content() | trim %}
+      <ol class="profile-page breadcrumb">
+        {% snippet 'snippets/home_breadcrumb_item.html' %}
+        {% block breadcrumb_content %}
+              <li><a href="{{ h.url_for('activity.dashboard') }}">{{ _('My account') }}</a></li>
+              <li class="active"><a>{{ _('My profile') }}</a></li>
+        {% endblock %}
+      </ol>
+    {% endif %}
+{% endblock %}
```

Comment:
Note that this template is used to show BOTH my personal profile and the profile of other users. So including "my profile" or "my account" isn't appropriate. e.g. visit [wac.minikube/](http://wac.minikube/user/dak_editor)

Reply:
If the current logged in user is the same as the owner of the profile then :  
![image](https://github.com/fjelltopp/ckanext-who-afro/assets/15812707/f271321c-09d7-4711-bacd-377b196815f8)
Otherwise : 
![image](https://github.com/fjelltopp/ckanext-who-afro/assets/15812707/9f8b6b72-e819-4e92-ac49-6f72e5a571bd)

To summarize:

The 'My account' from Figma has been reverted (sticking with Users as per default)

I have kept the style of the breadcrumb as designed in Figma

I have extended the header.html  (user/account_header.html) to accomodate the active style of 'My account' in the user profile page (first image above): 

```
    {% if c.userobj %}
        {{ h.build_nav_main(
            ('home.about', _('About')),
            ('dataset.search', _('Find Data')),
            ('insights.index', _('Insights'))
             ) }}
            <li class="{% if user_dict.name == g.userobj.name %}active my_account{% endif %}">
                <a href="{{ h.url_for('activity.dashboard') }}">{{ _('My account') }}</a>
            </li>
```